_search-plugins/searching-data/filtering-search-results.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: Filter results
+parent: Searching data
+nav_order: 21
+redirect_from:
+  - /opensearch/search/filter/
+---
+# Filtering search results in OpenSearch
```

Comment:
Shouldnt this be OpenSearch Dashboards? OpenSearch filtering is different and uses the API.

Reply:
Yes! This should be under the _dashboards directory.